src/app/shared/group.service.ts

```diff
@@ -202,10 +203,10 @@ export class GroupService {
     /**
      * Redeems an image group share token and returns an image group
      * @param token The image group share token
-     * @returns Observable with { success: boolean, group: ImageGroup }, although I'm not sure how to specify that in the typescript
+     * @returns Observable with { success: boolean, group: any }, although I'm not sure how to specify that in the typescript
      */
-    public redeemToken(token: string): Observable<{ success: boolean, group: ImageGroup }> {
-        return this.http.post<{ success: boolean, group: ImageGroup }>(
+    public redeemToken(token: string): Observable<{ success: boolean, group: any }> {
+        return this.http.post<{ success: boolean, group: any }>(
```

Comment:
What's the reason for removing `ImageGroup` as a type and replacing it with `any`?